<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1337</width>
    <height>996</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BrainStd</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#topFrame , #topFrameBackground{
border: none;
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #a6a6a6, stop: 0.08 #7f7f7f,
stop: 0.39999 #717171, stop: 0.4 #626262,
stop: 0.9 #4c4c4c, stop: 1 #333333);
}

#topFrame  QLabel{
color: white;
}

#topFrame  QCheckBox{
color: white;
}

#schemaFrame {
border: none;
background: white;
}

#schemaXMLframe {
border: none;
background: #7f7f7f;
}

#xmlOutFrame {
border: none;
background: #dfdfdf;
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="mainFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="topFrameBackground">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>41</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="topFrame">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>41</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>41</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCommandLinkButton" name="saveButton">
            <property name="maximumSize">
             <size>
              <width>41</width>
              <height>41</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../icons.qrc">
              <normaloff>:/new/prefix1/icons/save.png</normaloff>:/new/prefix1/icons/save.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>23</width>
              <height>23</height>
             </size>
            </property>
            <property name="autoRepeatDelay">
             <number>300</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCommandLinkButton" name="CreateNetworkButton">
            <property name="maximumSize">
             <size>
              <width>41</width>
              <height>41</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../icons.qrc">
              <normaloff>:/new/prefix1/icons/build.png</normaloff>:/new/prefix1/icons/build.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCommandLinkButton" name="playButton">
            <property name="maximumSize">
             <size>
              <width>41</width>
              <height>41</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../icons.qrc">
              <normaloff>:/new/prefix1/icons/Play-icon.png</normaloff>:/new/prefix1/icons/Play-icon.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="autoRepeatDelay">
             <number>300</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="timeLabel">
            <property name="minimumSize">
             <size>
              <width>180</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Fixedsys</family>
              <pointsize>15</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>00:00:000.000</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCommandLinkButton" name="actionsListButton">
            <property name="maximumSize">
             <size>
              <width>41</width>
              <height>41</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../icons.qrc">
              <normaloff>:/new/prefix1/icons/actions-list.png</normaloff>:/new/prefix1/icons/actions-list.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="autoRepeatDelay">
             <number>300</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCommandLinkButton" name="xmlButton">
            <property name="maximumSize">
             <size>
              <width>41</width>
              <height>41</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../icons.qrc">
              <normaloff>:/new/prefix1/icons/xml-tool-icon.png</normaloff>:/new/prefix1/icons/xml-tool-icon.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="autoRepeatDelay">
             <number>300</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCommandLinkButton" name="openFolderButton">
            <property name="minimumSize">
             <size>
              <width>128</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>128</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>11</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QCommandLinkButton{
color: white;
}
</string>
            </property>
            <property name="text">
             <string>Workspace</string>
            </property>
            <property name="icon">
             <iconset resource="../icons.qrc">
              <normaloff>:/new/prefix1/icons/save.png</normaloff>:/new/prefix1/icons/save.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>0</width>
              <height>10</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>    host name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="IPlineEdit">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>130</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>localhost</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>    port:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="PORTlineEdit">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>10000</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="tcpipLabel">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>  Connecting..</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frameToBeNamed">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="styleSheet">
             <string notr="true">QTabBar::tab {
  background: gray;
  color: white;
  padding-left: 20px;
  padding-right: 20px;
  height: 30px;
  min-width: 30px;
}

QTabBar::tab:selected {
  background: lightgray;
	color: black;
 }

#tabWidget::pane { border: 0; } 

</string>
            </property>
            <property name="currentIndex">
             <number>-1</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
      <zorder>topFrameBackground</zorder>
      <zorder>frameToBeNamed</zorder>
      <zorder>topFrame</zorder>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1337</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_new"/>
    <addaction name="action_load"/>
    <addaction name="action_save"/>
    <addaction name="action_save_as"/>
    <addaction name="action_export_network"/>
    <addaction name="separator"/>
    <addaction name="action_backup_schema"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionDocumentation_F1"/>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionClear_screen"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="action_grid"/>
   </widget>
   <widget class="QMenu" name="menuPlugins">
    <property name="title">
     <string>Windows</string>
    </property>
    <addaction name="actionXML_viewer"/>
    <addaction name="actionActions_list"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuLearning">
    <property name="title">
     <string>Learning</string>
    </property>
    <addaction name="actionActivated"/>
    <addaction name="actionLearning_window"/>
   </widget>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>Simulation</string>
    </property>
    <addaction name="action_create_network"/>
    <addaction name="actionRun"/>
    <addaction name="actionPause"/>
    <addaction name="actionStop_Un_load_network"/>
   </widget>
   <widget class="QMenu" name="menuModel">
    <property name="title">
     <string>Model</string>
    </property>
    <widget class="QMenu" name="menuBlocks">
     <property name="title">
      <string>Blocks</string>
     </property>
     <addaction name="action_add_new_block"/>
     <addaction name="action_remove_last_block"/>
    </widget>
    <widget class="QMenu" name="menuConnections">
     <property name="title">
      <string>Connections</string>
     </property>
     <addaction name="action_add_new_connection"/>
     <addaction name="action_remove_last_connection"/>
    </widget>
    <addaction name="menuBlocks"/>
    <addaction name="menuConnections"/>
    <addaction name="actionText_blocks"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuModel"/>
   <addaction name="menuSimulation"/>
   <addaction name="menuLearning"/>
   <addaction name="menuPlugins"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="action_load">
   <property name="text">
    <string>Load an existing network</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_export_network">
   <property name="text">
    <string>Export network</string>
   </property>
   <property name="toolTip">
    <string>Save neurons and connections in...</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Close the simulator</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionDocumentation_F1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Documentation</string>
   </property>
   <property name="toolTip">
    <string>Documentation</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionClear_screen">
   <property name="text">
    <string>Clear screen</string>
   </property>
   <property name="toolTip">
    <string>Delete the figure and the network that are currently running</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>Add new</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove last</string>
   </property>
  </action>
  <action name="actionActivated">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Activate learning</string>
   </property>
  </action>
  <action name="actionLearning_window">
   <property name="text">
    <string>Show learning window</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>Save the network in a brn file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_grid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Grid</string>
   </property>
  </action>
  <action name="actionXML_viewer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>XML viewer</string>
   </property>
  </action>
  <action name="actionActions_list">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Actions list</string>
   </property>
  </action>
  <action name="action_nemo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>NeMo</string>
   </property>
  </action>
  <action name="action_custom_simulator">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Custom Simulator</string>
   </property>
  </action>
  <action name="action_create_network">
   <property name="text">
    <string>Create Network</string>
   </property>
  </action>
  <action name="action_add_new_block">
   <property name="text">
    <string>Add new</string>
   </property>
  </action>
  <action name="action_remove_last_block">
   <property name="text">
    <string>Remove last</string>
   </property>
  </action>
  <action name="action_add_new_connection">
   <property name="text">
    <string>Add new</string>
   </property>
  </action>
  <action name="action_remove_last_connection">
   <property name="text">
    <string>Remove last</string>
   </property>
  </action>
  <action name="action_backup_schema">
   <property name="text">
    <string>Backup network</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionTCP_IP">
   <property name="text">
    <string>TCP/IP</string>
   </property>
  </action>
  <action name="action_new">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionStop_Un_load_network">
   <property name="text">
    <string>Stop (Un-load network)</string>
   </property>
  </action>
  <action name="actionText_blocks">
   <property name="text">
    <string>Text blocks</string>
   </property>
  </action>
  <action name="action_save_as">
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="toolTip">
    <string>Save the network in a new file</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
